client/view/workbook/info/Info.html

```diff
@@ -1,43 +1,81 @@
 <div class="container">
 
-    <header>
-        <h2>{workbook.name}</h2>
-    </header>
-
-    <main>
-        {#if forgotten}
-            {workbook.name} has been deleted.<br>
-            <button on:click="restoreWorkbook(key, workbook)">Restore</button>
-        {:else}
-            <button style="background-color: var(--red);" on:click="forgetAndPrepareForUndo(key)">Remove "{workbook.name}" from the home page</button>
+	<header>
+		<h2>{workbook.name}</h2>
+	</header>
+
+	<main>
+		{#if forgotten}
+			{workbook.name} has been deleted.<br>
+			<button on:click="restoreWorkbook(key, workbook)">Restore</button>
+		{:else}
+			
+			<p>This workbook has { workbook.sheets.length } sheet{#if workbook.sheets.length !== 1}s{/if}.</p>
+
+			<p>
+				{#if workbook.sheets.length == 0}
+					There are no sheets in this workbook.
+				{:elseif workbook.sheets.length == 1}
+					<a href="{ asr.makePath('workbook.sheet.memorize', { key, sheetId: workbook.sheets[0].id }) }">{workbook.sheets[0].name}</a>
+				{:else}
+					{#each workbook.sheets as sheet}
+						{#if sheet.id == secondToLastSheetId }
+							<a href="{ asr.makePath('workbook.sheet.memorize', { key, sheetId: sheet.id }) }">{sheet.name}</a> and&nbsp;
+						{:elseif sheet.id == lastSheetId}
+							<a href="{ asr.makePath('workbook.sheet.memorize', { key, sheetId: sheet.id }) }">{sheet.name}</a>.
+						{:else}
+							<a href="{ asr.makePath('workbook.sheet.memorize', { key, sheetId: sheet.id }) }">{sheet.name}</a> ,&nbsp;
+						{/if}
+					{:else}
+						This workbook is empty.
+					{/each}
+				{/if}
+			</p>
+		
+			<p>
+				<a href="https://docs.google.com/spreadsheets/d/{key}" target="_blank">View workbook on Google Sheets</a>
```

Comment:
We should set [`rel`](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types) to `noopener` and `external`, too.